core/new-gui/src/environments/environment.default.ts

```diff
@@ -36,12 +36,12 @@ export const defaultEnvironment = {
   /**
    * whether user system is enabled
    */
-  userSystemEnabled: false,
+  userSystemEnabled: true,
 
   /**
    * whether local login is enabled
    */
-  localLogin: true,
+  localLogin: false,
```

Comment:
shall we revert the default value changes?

Reply:
I pushed this by mistake.